packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemNextDirectory.kt

```diff
@@ -0,0 +1,13 @@
+package expo.modules.filesystem.next
+
+import java.io.File
+
+class FileSystemNextDirectory(file: File) : FileSystemNextPath(file) {
+  fun validatePath() {
+// Kept empty for now, but can be used to validate if the path is a valid directory path.
```

Comment:
We need this implemented on iOS, and any checks on URL validity need to be done here as you can't throw from constructors (for now?)

Reply:
What happens if you throw from the constructor? I would assume it will crash the app. If yes, that should be a quick fix.

Reply:
It does nothing on iOS – I don't think it's a super quick fix 😅